services/ui-src/src/components/fields/TextField.tsx

```diff
@@ -34,11 +34,12 @@ export const TextField = ({
     const fieldValue = form.getValues(name);
     if (fieldValue) {
       setDisplayValue(fieldValue);
+      form.setValue(name, fieldValue);
     }
     // else set hydrationValue or defaultValue as display value
     else if (hydrationValue) {
       setDisplayValue(hydrationValue);
-      form.setValue(name, hydrationValue);
+      form.setValue(name, hydrationValue, { shouldValidate: true });
```

Comment:
Comment: Assuming this is the fix -- it seems like the issue is actually just that the form was not revalidating the field after it's hydrated, which makes sense.

Reply:
Yup! Real easy one